user/developer.md

```diff
@@ -27,6 +27,24 @@ We've created an [API Explorer](https://developer.travis-ci.org/) that closely
 integrates with V3, updating automatically when new endpoints are added, and
 includes a useful tool for exploring endpoints.
 
+## API V2.1
+
+We've released an update to the Travis CI API V2, which is API V2.1. This update allows users of Travis CI for private projects to make certain build information available through the API.
+
+For users of Travis CI for public repositories there is no change.
```

Comment:
Should we say something like: "There is no change for our API for Open Source projects built at api.travis-ci.org" or similar?

Reply:
Fixed